### ISSUE TITLE
Unnamed anonymous type variable naming issue

### ISSUE THREAD

**Issue by MaceWindu** [agileobjects/ReadableExpressions#74]
Reproduced at least with 3.2.2

![image](https://user-images.githubusercontent.com/1099238/82731319-428b2000-9d06-11ea-9291-ca7085286780.png)

Offender:
![image](https://user-images.githubusercontent.com/1099238/82731346-6fd7ce00-9d06-11ea-88b6-8feb7bbf5730.png)

```cs
(qr, dr) => ((
    qr,
    dctx,
    rd,
    expr,
    ps,
    preamble) =>
{
    var ldr = (MySqlDataReader)dr;
    var anonymousType_Span class="kw">bool</span>_Span class="kw">int</span>_Span class="kw">int</span> = new { IsGroping = (ldr.IsDBNull(0) ? 0 : (int)ldr.GetInt64(0)) == 1, Id1 = ldr.GetInt32(1), Count = (int)ldr.GetInt64(2) };

    return anonymousType_Span class="kw">bool</span>_Span class="kw">int</span>_Span class="kw">int</span>;
}).Invoke(
    qr,
    qr.DataContext,
    dr,
    qr.Expression,
    qr.Parameters,
    qr.Preambles)
```

**Comment by SteveWilkes** [comment]
Thanks for letting me know - I'll get this fixed :)

**Comment by SteveWilkes** [comment]
This is fixed in the latest v3.3 release branch code - release to follow. Thanks!

**Comment by SteveWilkes** [comment]
This is fixed in [NuGet package v2.5](https://nuget.org/packages/AgileObjects.ReadableExpressions), and [visualizers v3.3](https://marketplace.visualstudio.com/items?itemName=vs-publisher-1232914.ReadableExpressionsVisualizers).

Thanks again!

Steve
